libs/watt/src/lib/components/dropdown/watt-dropdown.component.ts

```diff
@@ -67,33 +67,19 @@ import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
   ],
 })
 export class WattDropdownComponent implements ControlValueAccessor, OnInit {
-  /**
-   * @ignore
-   */
   private _destroyRef = inject(DestroyRef);
```

Comment:
suggestion: consider reverting the changes.

It looks like the comment works. Here's before and after when running Storybook locally. I haven't tested with all properties, but it seems like adding the comment removes properties from Storybook.

**Notice the "Properties" section**

Before (with comment present)

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/76ad4d01-ee36-4a16-8b06-16316ab60e6a)

After (with comment removed)

![image](https://github.com/Energinet-DataHub/greenforce-frontend/assets/1096332/2ef3fdf9-8cd5-4d20-9dcc-1fa034d3e0b9)